landing-pages/site/assets/scss/_buttons.scss

```diff
@@ -48,10 +48,16 @@ button {
     svg {
       height: $icon-height;
       width: auto;
-      padding-right: 15px;
+    }
+
+    @media (max-width: 1279px) {
+      svg {
+        padding-right: 15px;
+      }
     }
```

Comment:
I think it would be better to address the root cause of the issue first.

The root cause seems to be in the following part of the code:
https://github.com/apache/airflow-site/blob/48c9a3bed01af394ffde6ca9c573d3d1eaa9584c/landing-pages/site/assets/scss/_base-layout.scss#L80-L86

If we improve this section, we may be able to avoid adding additional media queries.